WordPress/Classes/ViewRelated/Blog/BlogDetailsViewController.m

```diff
@@ -435,12 +435,19 @@ - (BlogDetailsSection *)publishTypeSectionViewModel
                                                  callback:^{
                                                      [weakSelf showMediaLibrary];
                                                  }]];
+    
+    [rows addObject:[[BlogDetailsRow alloc] initWithTitle:NSLocalizedString(@"Portfolio", @"Noun. Title. Links to the user's portfolio projects.")
+                                                    image:[Gridicon iconOfType:GridiconTypeImage]
```

Comment:
Not sure if the `image` should be the same we've got for Media. Looks a bit odd.

Not sure what the original issue is (Can't find mockups // the Portfolio option anywhere in dotcom). @mattmiklic mind chiming in? (Perhaps Gridicons > Folder?)

Reply:
Yep! Folder for portfolio (and quote for testimonials, if we happen to implement those). You can turn on Portfolio and Testimonials for any site by going to Settings > Writing > Content types. 

<img width="302" alt="screen shot 2017-10-13 at 1 50 37 pm" src="https://user-images.githubusercontent.com/1396552/31559158-a98c386c-b01d-11e7-85f8-d193ac14ad99.png">

Reply:
Thanks for the superquick response!

Reply:
@mattmiklic , @jleandroperez thanks for the comments. I wanted to make this PR even though I was waiting for an answer from @aerych on this. He explained where the icons are. Gonna update in the next PR. Also, @aerych mentioned that my initial PRs would be labeled with "DO_NOT_MERGE" is that a label I can add?

Reply:
Sorry to keep you waiting.  I've been a bit disrupted due to flights. 
I've added the label :)